roles/edpm_bootstrap/meta/argument_specs.yml

```diff
@@ -31,6 +31,8 @@ argument_specs:
           - iproute-tc
           - ksmtuned
           - systemd-container
+          - crypto-policies-scripts
+          - grubby
```

Comment:
I have proposed adding these packages to the edpm image also https://github.com/openstack-k8s-operators/edpm-image-builder/pull/24